src/main/java/edu/ie3/datamodel/io/processor/ProcessorProvider.java

```diff
@@ -235,20 +235,36 @@ public ProcessorProvider(
    *
    * @return a collection of all existing processors
    */
-  private Collection<EntityProcessor<? extends UniqueEntity>> allEntityProcessors() {
-
+  public static Collection<EntityProcessor<? extends UniqueEntity>> allEntityProcessors() {
```

Comment:
```suggestion
  public static Collection<EntityProcessor<UniqueEntity>> allEntityProcessors() {
```

Please remove all those code smells, where a wildcard is part of the return type (See sonarqube analysis).

Reply:
As decided with @johanneshiry, this doesn't resolve. Marked the issue in SonarQube accordingly.